tutorials/run-swift-vapor-on-google-cloud/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ibmcom/swift-ubuntu:latest
+LABEL Description="Docker image for Swift + Vapor on GAE Flex."
+
+# Expose default port for GAE
+EXPOSE 8080
+
+# Copy sources
+RUN mkdir /root/VaporGAE
```

Comment:
I think you can boil all of this (before the build) down to:

    COPY . /app/

No need to mkdir, or pick and choose individual files to copy.  Creating an `app` directory in the container is more consistent with our runtimes too.